### ISSUE TITLE
## Issue: Unable to change OpenAPI key after pasting a wrong key

### ISSUE THREAD

**Issue by Wladefant** [roblourens/chat-agent-dalle#4]
### Description
After mistakenly pasting an incorrect OpenAPI key, I am unable to change it to the correct one. The system does not provide an option to update the key.

### Steps to Reproduce
1. Copy an incorrect OpenAPI key.
2. Paste the incorrect key into the designated field.
3. Attempt to change the key to the correct one.
4. Notice that there is no option or functionality to update the key.

### Expected Behavior
The system should allow users to update the OpenAPI key after pasting an incorrect one.

Assignees: 
Labels: 
Milestone:

**Comment by roblourens** [comment]
Good point. I'm not sure why you closed these. But I would take a PR that adds a command for this.

**Comment by Wladefant** [comment]
Should have been an issue in my fork, somehow the github extension created it here, but I reopened it

**Comment by Wladefant** [comment]
I am a bit new to extension development. Can you tell how to reset an extension? The problem yesterday was that the extension host seemed to remember everything and the key too and I had to manually change the code and delete the openai key, before the reverting the changes once again. Is there a way to just reset everything related to the extension?

**Comment by roblourens** [comment]
It's probably not possible (unless you change the extension ID), so we should add a command to have the extension manually reset the stored key.

**Comment by Wladefant** [comment]
I see. Could you also tell me when the proposed api is planned to go public?

**Comment by roblourens** [comment]
Not sure, hopefully in the next few months

**Comment by undesicimo** [comment]
Haven't heard about the proposed api lately, any news?

**Comment by roblourens** [comment]
Follow https://github.com/microsoft/vscode/issues/199908 for updates
